core/command/reference.js

```diff
@@ -22,6 +23,7 @@ module.exports = {
         return createBitmaps(config, true);
       }).then(function () {
         console.log('\nRun `$ backstop test` to generate diff report.\n');
+        return executeCommand('_engineErrors', config);
```

Comment:
you should be able to just run a plain old module. like here...   https://github.com/garris/BackstopJS/blob/master/core/command/test.js#L10